### PR TITLE
Fix panic when expr invalid

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>
Issue: #19617

### CODE REVIEW

**Review by congqixia — APPROVED** [review]
/lgtm

File: internal/proxy/plan_parser.go [hunk 1]
```diff
@@ -858,7 +858,7 @@ func (pc *parserContext) handleExpr(nodeRaw *ant_ast.Node) (*planpb.Expr, error)
 	case *ant_ast.BinaryNode:
 		return pc.handleBinaryExpr(node)
 	default:
-		return nil, fmt.Errorf("unsupported node (%s)", node.Type().String())
+		return nil, fmt.Errorf("unsupported node")
```
Comment (xiaofan-luan) [inline internal/proxy/plan_parser.go:861]: should print the original expression?

Comment (cydrain) [reply internal/proxy/plan_parser.go:861]: original expr has been printed out in upper level

**Comment by cydrain** [comment]
rerun ut

**Pull request merged**